README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
The semicolon seems misplaced. Is it intended to be part of the code, or is it a typo?